gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineIdea.groovy

```diff
@@ -68,6 +68,21 @@ class BaselineIdea extends AbstractBaselinePlugin {
             moveProjectReferencesToEnd(ideaModuleModel)
         }
 
+        // Suggest and configure the "save actions" plugin if Palantir Java Format is turned on.
+        if (project == project.rootProject) {
```

Comment:
it seems like nearly the whole plugin should be a no-op if not applied to the root. perhaps we can restructure this so its a little more clear

Reply:
restructured